src/pages/index.jsx

```diff
@@ -1,87 +1,60 @@
-              helperTextInvalid="Value must be an URL"
-              helperTextInvalidIcon={<ExclamationCircleIcon />}
-              className="label"
-              fieldId="url"
-              validated={validate()}
-            >
-              <InputGroup>
-                <TextInput
-                  type="text"
-                  value={value}
-                  validated={validate()}
-                  required={true}
-                  placeholder="https://github.com/org/repo"
-                  onChange={handleValueChange}
-                  aria-label="text input example"
-                  id="url"
-                  style={{ minWidth: '20vw', verticalAlign: 'middle' }}
-                />
-                <Button variant="primary" isLoading={isSubmitted} onClick={handleSubmit} isDisabled={pristine || validate() !== 'success'}>
-                  {(!isSubmitted && <ArrowRightIcon />) || <>Submitting</>}
-                </Button>
-              </InputGroup>
-            </FormGroup>
-          </Form>
-        </CardBody>
-      </Card>
-    </Bullseye>
-  );
-};
+import Form from '../components/Form';
+import Layout from '../components/Layout';
+import Header from '../components/Header';
+import Meteors from '../components/Meteors';
+
+const Index = () => (
+  <Layout>
+    <Head>
+      <title>Meteor</title>
+    </Head>
+    <Header>
+      <Bullseye>
+        <TextContent style={{ color: 'unset' }}>
+          <Text component={TextVariants.h1}>Meteor</Text>
+          <Text component={TextVariants.p}>Take your Jupyter Notebooks for a spin and show your impact.</Text>
+        </TextContent>
+      </Bullseye>
+    </Header>
+    <PageSection style={{ backgroundColor: 'transparent' }}>
+      <Bullseye>
+        <Form />
+      </Bullseye>
+    </PageSection>
+    <PageSection isFilled>
+      <Flex alignItems={{ default: 'alignItemsCenter' }} justifyContent={{ default: 'justifyContentCenter' }}>
+        <FlexItem style={{ maxWidth: '30%' }}>
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptate explicabo, distinctio eveniet quam voluptates rerum soluta assumenda
+          nihil laborum consequuntur eos magnam repellendus, maxime cum iusto illo? Maiores fugit autem ipsam ab ullam laborum, magni expedita
+          consectetur sint quas, enim mollitia libero hic! Eos, obcaecati? Sapiente corrupti pariatur et fuga.
+        </FlexItem>
+        <Divider isVertical />
+        <FlexItem style={{ maxWidth: '30%' }}>
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptates esse facilis a! Suscipit esse facere delectus quo pariatur autem
+          aliquid, quos dignissimos, ex fugit laborum. Dignissimos harum error ab quaerat atque non blanditiis cupiditate iusto minima sunt velit
+          praesentium illum deserunt obcaecati totam, dolores voluptatibus, porro earum voluptates odit ut? Sed molestiae amet, corrupti dolorem
+          laboriosam corporis adipisci aspernatur.
```

Comment:
@oindrillac @isabelizimm @margarethaley @durandom

can you help me with some text for this? I need some outline explaining and describing the Meteor project and it's integrations in here.

Reply:
1. Use the same level of information as on https://mybinder.org/ - you can almost copy the text there, but with s/Docker/Container/ etc
2. Make sure to link back to this repo

Reply:
Sure thing! I can put that in a different PR, if that's easier?

Reply:
@isabelizimm yes! let's do it that way! Lets merge this one then and you can open a follow up. :slightly_smiling_face: